src/accordion/accordion.directive.ts

```diff
@@ -32,20 +33,22 @@ let nextId = 0;
  *
  * @since 14.1.0
  */
-@Directive({
+@Component({
```

Comment:
This reverts #4502

Reply:
Indeed !
I have added a public `elementRef` reference in order to help retrieve the native element with template variables.